tutorials/pincell_multiphysics/pincell.py

```diff
@@ -82,7 +74,7 @@
 # Create an initial uniform spatial source distribution over fissionable zones
 lower_left = (-pitch, -pitch, 0.0)
 upper_right = (pitch, pitch, height)
-uniform_dist = openmc.stats.Box(lower_left, upper_right, only_fissionable = True)
+uniform_dist = openmc.stats.Box(lower_left, upper_right)
 
 settings.source = openmc.IndependentSource(space=uniform_dist)
```

Comment:
```suggestion
settings.source = openmc.IndependentSource(space=uniform_dist, constraints={'fissionable' : True})
```

Reply:
if you accept this and the other change related to constraints, please re-update the XML files too